packages/wrangler/templates/middleware/loader-sw.ts

```diff
@@ -0,0 +1,186 @@
+import { Awaitable, Dispatcher, Middleware, __facade_invoke__ } from "./common";
+export { __facade_register__, __facade_registerInternal__ } from "./common";
+
+const __FACADE_EVENT_TARGET__ = new EventTarget();
+
+function __facade_isSpecialEvent__(type: string) {
+	return type === "fetch" || type === "scheduled";
+}
+(globalThis as any).__facade_addEventListener__ = function (
```

Comment:
You can widen the `globalThis` type rather than casting `any` to shoehorn a new method. 
```suggestion
declare global {
var __facade_addEventListener__: <whatever type>
... other methods 
}

function __facade_isSpecialEvent__(type: string) {
	return type === "fetch" || type === "scheduled";
}
(globalThis as any).__facade_addEventListener__ = function (
```

Reply:
Resolved in https://github.com/cloudflare/wrangler2/pull/1735/commits/388c00d8d8ca9dc32a9343174fce2468013ffe23